RFE: is it possible to start making github releases?🤔

On create github release entry is created email notification to those whom have set in your repo the web UI Watch->Releases.
gh release can contain additional comments (li changelog) or additional assets like release tar balls (by default it contains only assets from git tag) however all those part are not obligatory.
In simplest variant gh release can be empty because subiekt of the sent email contains git tag name.

I'm asking because my automation process uses those email notifications by trying to make preliminary automated upgrades of building packages, which allows saving some time on maintaining packaging procedures.
Probably other people may be interested to be instantly informed about release new version as well.

Documentation and examples of generate gh releases:
https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository
https://cli.github.com/manual/gh_release_upload/
https://github.com/marketplace/actions/github-release
https://pgjones.dev/blog/trusted-plublishing-2023/
https://github.com/jbms/sphinx-immaterial/issues/281#issuecomment-1700933026
tox target to publish on pypi and make gh release https://github.com/jaraco/skeleton/blob/928e9a86d61d3a660948bcba7689f90216cc8243/tox.ini#L42-L58

The current release fails with this error:

```
Run elgohr/Github-Release-Action@v4
Run $GITHUB_ACTION_PATH/entrypoint.sh
/home/runner/work/_actions/elgohr/Github-Release-Action/v4/entrypoint.sh: 2: hub: not found
Error: Process completed with exit code 127.
```
https://github.com/collective/icalendar/actions/runs/6748355950/job/18346549191

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -122,8 +122,4 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: create release
-        uses: elgohr/Github-Release-Action@v5
-        env:
-          GH_TOKEN: ${{ github.token }}
-        with:
-          title: ${{ github.ref_name }}
+        uses: ncipollo/release-action@v1

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -124,6 +124,6 @@ jobs:
       - name: create release
         uses: elgohr/Github-Release-Action@v5
         env:
-          GITHUB_TOKEN: ${{ secrets.RELEASE_TOKEN }}
+          GH_TOKEN: ${{ github.token }}
         with:
           title: ${{ github.ref_name }}

--- a/CHANGES.rst
+++ b/CHANGES.rst
@@ -14,7 +14,7 @@ Breaking changes:
 
 New features:
 
-- ...
+- Create GitHub releases for each tag.
 
 Bug fixes:
 

